modules/notes/assets/js/hooks/ui/panel/state-ready/notes-add-panel-menu-item.js

```diff
@@ -0,0 +1,29 @@
+export class NotesAddPanelMenuItem extends $e.modules.hookUI.After {
+	getCommand() {
+		return 'panel/state-ready';
+	}
+
+	getId() {
+		return 'notes-add-panel-menu-item';
+	}
+
+	apply() {
+		elementor.modules.layouts.panel.pages.menu.Menu.addItem( {
+			name: 'notes',
+			icon: 'eicon-commenting-o',
+			title: __( 'Notes', 'elementor-pro' ),
+			callback() {
+				elementor.promotion.showDialog( {
+					headerMessage: __( 'Notes', 'elementor' ),
+					message: __( 'With Notes, teamwork gets even better. Stay in sync with comments, feedback & more on your website.', 'elementor' ),
+					top: '-3',
+					inlineStart: '+10',
+					element: this.$el,
+					actionURL: 'https://go.elementor.com/go-pro-notes/',
+				} );
+			},
+		}, 'navigate_from_page', 'finder' );
+	}
+}
+
+export default NotesAddPanelMenuItem;
```

Comment:
Why not at the beginning?

Reply:
When using Named export ( `export class NAME {` ), instead of Default export ( `export default class NAME {` ), you wrap the exports in braces in the index.js files by thus allowing `import * as hooks` in e-component.js.